Make issues: debug is broken, run is impossible to control-C out of

make debug- is broken. `make debug-libfuzzer-zlib_zlib_uncompress_fuzzer` fails with this error:
```
docker run \                                                                                                                                                                                                       
--cpus=1 \                                                                                                                                                                                                         
--cap-add SYS_NICE \                                                                                                                                                                                               
--cap-add SYS_PTRACE \                                                                                                                                                                                             
-e FUZZ_OUTSIDE_EXPERIMENT=1 \                                                                                                                                                                                     
-e FORCE_LOCAL=1 \                                                                                                                                                                                                 
-e TRIAL_ID=1 \                                                                                                                                                                                                    
-e FUZZER=libfuzzer \                                                                                                                                                                                              
-e BENCHMARK=zlib_zlib_uncompress_fuzzer \                                                                                                                                                                         
-e FUZZ_TARGET=zlib_uncompress_fuzzer \                                                                                                                                                                            
-entrypoint "/bin/bash" \                                                                                                                                                                                          
-it gcr.io/fuzzbench/runners/libfuzzer/zlib_zlib_uncompress_fuzzer                                                                                                                                                 
docker: invalid reference format.                                                                                                                                                                                  
See 'docker run --help'.                                                                                                                                                                                           
make: *** [docker/generated.mk:26568: debug-libfuzzer-zlib_zlib_uncompress_fuzzer] Error 125
```

make `run-libfuzzer-zlib_zlib_uncompress_fuzzer` runs forever and cannot be Ctrl-Ced out of.

--- a/docker/generate_makefile.py
+++ b/docker/generate_makefile.py
@@ -58,11 +58,11 @@ def _print_makefile_run_template(image):
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
-            print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+            print('\t--entrypoint "/bin/bash" \\\n\t', end='')
         else:
             print('\t', end='')
 
-        print(os.path.join(BASE_TAG, image['tag']))
+        print('-it ' + os.path.join(BASE_TAG, image['tag']))
         print()
 
 

--- a/docker/generate_makefile.py
+++ b/docker/generate_makefile.py
@@ -58,11 +58,13 @@ def _print_makefile_run_template(image):
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
-            print('\t--entrypoint "/bin/bash" \\\n\t', end='')
+            print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+        elif run_type == 'run':
+            print('\t-it ', end='')
         else:
             print('\t', end='')
 
-        print('-it ' + os.path.join(BASE_TAG, image['tag']))
+        print(os.path.join(BASE_TAG, image['tag']))
         print()
 
 

